### PR TITLE
First action scripts.

### PR DESCRIPTION
**Pull request by jvanz**
Adds the first version of the action used to create a Kuberwarden cluster.

### CODE REVIEW

**Comment by jvanz** [comment]
Related to https://github.com/kubewarden/kubewarden-controller/issues/125

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (jvanz) [inline LICENSE]: Do we have any default license to be used?

Comment (flavio) [reply LICENSE]: I would stick with Apache 2, like we do with the other code we created

**Review by viccuad — APPROVED** [review]
Approving. 
I'm totally ok with this implementation if a majority agrees, but I still have the same concerns listed in https://github.com/kubewarden/kubewarden-controller/pull/127#pullrequestreview-816652649. From previous CI work, I think it's a good idea to future-proof by allowing to partially run them against any cluster and stack, outside of GHA.
In addition, we have a bunch of shellscript there (and we may have other things in the near future). I would prefer if those snippets where in their files, and we had linting for them (shellcheck, etc).

**Pull request merged**